Unaccountable-Magpie/BangazonAPI2
# Bs models3

Purpose:
Added the "IsDeleted Boolean"


How it fits in context of the project:

Allows objects to be "fake" deleted

Specific Feature affected:

Models/computers

How to test (Be thorough!):

1. Open models folder

2. Open computers

3. Check if the new line has been added
